common/shield_exceptions.cc

```diff
@@ -38,46 +38,6 @@ bool IsBlockedResource(const GURL& gurl) {
                      });
 }
 
-bool IsWhitelistedCookieException(const GURL& firstPartyOrigin,
-    const GURL& subresourceUrl, bool allow_google_auth) {
-  // Note that there's already an exception for TLD+1, so don't add those here.
-  // Check with the security team before adding exceptions.
-
-  // 1st-party-INdependent whitelist
-  std::vector<URLPattern> fpi_whitelist_patterns = {};
-  if (allow_google_auth) {
-    fpi_whitelist_patterns.push_back(URLPattern(URLPattern::SCHEME_ALL,
-        "https://accounts.google.com/o/oauth2/*"));
-  }
-  bool any_match = std::any_of(fpi_whitelist_patterns.begin(),
-      fpi_whitelist_patterns.end(),
-      [&subresourceUrl](const URLPattern& pattern) {
-        return pattern.MatchesURL(subresourceUrl);
-      });
-  if (any_match) {
-    return true;
-  }
-
-  // 1st-party-dependent whitelist
-  static std::map<GURL, std::vector<URLPattern> > whitelist_patterns = {
-    {
-      GURL("https://www.sliver.tv/"),
-      std::vector<URLPattern>({URLPattern(URLPattern::SCHEME_ALL,
-            "https://*.thetatoken.org:8700/*")})
```

Comment:
@bridiver This exception needs to be in place to unbreak a partner site (sliver.tv).  I don't see it added back in elsewhere in the PR, so just wanted to flag it and make sure its carried over

Reply:
must be a rebase issue, I'll move it to the new location

Reply:
updated https://github.com/brave/brave-core/pull/3130/files#diff-da6fb1ceb9261944ea46907846888791R57